conversions/octal_to_hexadecimal.py

```diff
@@ -0,0 +1,73 @@
+def octal_to_hex(octal: str) -> str:
+    """
+    Convert an Octal number to Hexadecimal number.
+
+    >>> octal_to_hex("100")
+    '0X40'
+    >>> octal_to_hex("235")
+    '0X9D'
+    >>> octal_to_hex(17)
+    Traceback (most recent call last):
+        ...
+    TypeError: Expected a string as input
+    >>> octal_to_hex("Av")
+    Traceback (most recent call last):
+        ...
+    ValueError: Not a Valid Octal Number
+    >>> octal_to_hex("")
+    Traceback (most recent call last):
+        ...
+    ValueError: Empty string was passed to the function
+
+    For more information: https://en.wikipedia.org/wiki/Octal
+    """
```

Comment:
```suggestion
    """
    Convert an Octal number to Hexadecimal number.
    For more information: https://en.wikipedia.org/wiki/Octal

    >>> octal_to_hex("100")
    '0X40'
    >>> octal_to_hex("235")
    '0X9D'
    >>> octal_to_hex(17)
    Traceback (most recent call last):
        ...
    TypeError: Expected a string as input
    >>> octal_to_hex("Av")
    Traceback (most recent call last):
        ...
    ValueError: Not a Valid Octal Number
    >>> octal_to_hex("")
    Traceback (most recent call last):
        ...
    ValueError: Empty string was passed to the function
    """
```

Place all explanation at the top rather than separating it between the doctests